packages/docusaurus-theme-classic/src/theme/SkipToContent/index.tsx

```diff
@@ -5,35 +5,43 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, {useRef, useEffect} from 'react';
+import {useLocation} from '@docusaurus/router';
 
 import styles from './styles.module.css';
 
+function programmaticFocus(el) {
+  el.setAttribute('tabindex', '-1');
+  el.focus();
+  setTimeout(() => el.removeAttribute('tabindex'), 1000);
+}
```

Comment:
didn't know this was possible, nice trick
is it safe in all cases to use? (seems to work for Chrome + Safari at least)

Reply:
I think yes, it's safe.